From e24ad92e6f13d6840be8e8b35e193fd723a55728 Mon Sep 17 00:00:00 2001
From: Alex Perrin <44126061+AlexPerrin@users.noreply.github.com>
Date: Sat, 17 Oct 2026 17:58:51 -0400
Subject: [PATCH 1/2] Lumbridge and Edgeville general stores open their own
 stock

Lumbridge's shopkeeper opened inv.generalshop1 (the Al Kharid stock, no spade or knife)
and Edgeville's opened inv.generalshop5 (the Lumbridge stock). Lumbridge now opens
generalshop5 (spade and knife, as on the wiki; X Marks the Spot sends players there for a
spade) and Edgeville generalshop2.
---
 .../content/areas/city/edgeville/npcs/EdgevilleGeneralStore.kt  | 2 +-
 .../org/rsmod/content/areas/city/lumbridge/npcs/GeneralStore.kt | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/content/areas/city/edgeville/src/main/kotlin/org/rsmod/content/areas/city/edgeville/npcs/EdgevilleGeneralStore.kt b/content/areas/city/edgeville/src/main/kotlin/org/rsmod/content/areas/city/edgeville/npcs/EdgevilleGeneralStore.kt
index 25affa4d8..f90630a2e 100644
--- a/content/areas/city/edgeville/src/main/kotlin/org/rsmod/content/areas/city/edgeville/npcs/EdgevilleGeneralStore.kt
+++ b/content/areas/city/edgeville/src/main/kotlin/org/rsmod/content/areas/city/edgeville/npcs/EdgevilleGeneralStore.kt
@@ -30,7 +30,7 @@ class EdgevilleGeneralStore @Inject constructor(private val shops: Shops) : Plug
     }
 
     private fun Player.openGeneralStore(npc: Npc) {
-        shops.open(this, npc, "Edgeville General Store", "inv.generalshop5")
+        shops.open(this, npc, "Edgeville General Store", "inv.generalshop2")
     }
 
     private companion object {
diff --git a/content/areas/city/lumbridge/src/main/kotlin/org/rsmod/content/areas/city/lumbridge/npcs/GeneralStore.kt b/content/areas/city/lumbridge/src/main/kotlin/org/rsmod/content/areas/city/lumbridge/npcs/GeneralStore.kt
index fe28669ce..abdd237e2 100644
--- a/content/areas/city/lumbridge/src/main/kotlin/org/rsmod/content/areas/city/lumbridge/npcs/GeneralStore.kt
+++ b/content/areas/city/lumbridge/src/main/kotlin/org/rsmod/content/areas/city/lumbridge/npcs/GeneralStore.kt
@@ -20,7 +20,7 @@ class GeneralStore @Inject constructor(private val shops: Shops) : PluginScript(
     }
 
     private fun Player.openGeneralStore(npc: Npc) {
-        shops.open(this, npc, "Lumbridge General Store", "inv.generalshop1")
+        shops.open(this, npc, "Lumbridge General Store", "inv.generalshop5")
     }
 
     private suspend fun ProtectedAccess.shopDialogue(npc: Npc) =
-- 
2.43.0

